sys/net/application_layer/dhcpv6/client.c

```diff
@@ -482,7 +483,8 @@ static void _parse_advertise(uint8_t *adv, size_t len)
                     dhcpv6_opt_ia_pd_t *ia_pd = (dhcpv6_opt_ia_pd_t *)opt;
                     unsigned pd_t1, pd_t2;
                     uint32_t ia_id = byteorder_ntohl(ia_pd->ia_id);
-                    size_t ia_pd_len = byteorder_ntohs(ia_pd->len);
+                    size_t ia_pd_len = byteorder_ntohs(ia_pd->len) - 
```

Comment:
[Travis reports a whitespace error](https://travis-ci.org/github/RIOT-OS/RIOT/builds/661619176#L157-L161) here.

Reply:
(please squash directly)

Reply:
I should always perform `make static-test` before pushing.